dataloader.py

```diff
@@ -89,13 +97,27 @@ def __init__(self, opt):
         for split in self.iterators.keys():
             self._prefetch_process[split] = BlobFetcher(split, self, split=='train')
             # Terminate the child process when the parent exists
+
         def cleanup():
             print('Terminating BlobFetcher')
             for split in self.iterators.keys():
                 del self._prefetch_process[split]
+
         import atexit
         atexit.register(cleanup)
 
+        toc_filename = os.path.join(self.input_fc_dir, 'dataset_fc.toc')
+        with open(toc_filename, 'rt') as f:
+            self.fc_toc = {l.split()[0]: int(l.split()[1])
+                           for l in f.readlines()}
```

Comment:
Is it simpler just saving the offsets in a pickle file or a json file?

Reply:
e3b7c415b3ebded1356ab7de1550332b8a85bfa1